### ISSUE TITLE
Load failure: different kinds of errors 

### ISSUE THREAD

**Issue by ModernMats** [huggingface/neuralcoref#116]
Hi, trying to install NeuralCoref but do not succeed however I try to. I'm on a macOS Mojave. Tried in different environments, conda with py2.7 and py3.6, clean installs with virtualenvironment 2.7 and 3.6 and the two different approaches mentioned [here.](https://github.com/huggingface/neuralcoref) 

**First approach** (still py3.6), loading with:
`nlp2 = spacy.load('./venv3/lib/python3.6/site-packages/en_coref_md')`

I get:
`KeyError: "[E002] Can't find factory for 'neuralcoref'. This usually happens when spaCy calls 'nlp.create_pipe' with a component name that's not built in - for example, when constructing the pipeline from a model's meta.json. If you're using a custom component, you can write to 'Language.factories['neuralcoref']' or remove it from the model meta and add it via 'nlp.add_pipe' instead."`

**Second approach** with py3.6, clean pip install 
(`pip install https://github.com/huggingface/neuralcoref-models/releases/download/en_coref_md-3.0.0/en_coref_md-3.0.0.tar.gz` as suggested in docs)

in a virtual environment, loading with:
`import spacy`
`import en_coref_md`
`nlp = en_coref_md.load()`

I get the following error:
`    from .neuralcoref import NeuralCoref`
`  File "cymem.pxd", line 1, in init en_coref_md.neuralcoref.neuralcoref`
`ValueError: cymem.cymem.Pool has the wrong size, try recompiling. Expected 64, got 48`

**Next, the same install but in a clean py2.7 environment, both approaches**. 
Trying to load with the first approach:
`nlp2 = spacy.load('./venv/lib/python2.7/site-packages/en_coref_md/')`
I also get:
`KeyError: u"[E002] Can't find factory for 'neuralcoref'. This usually happens when spaCy calls 'nlp.create_pipe' with a component name that's not built in - for example, when constructing the pipeline from a model's meta.json. If you're using a custom component, you can write to 'Language.factories['neuralcoref']' or remove it from the model meta and add it via 'nlp.add_pipe' instead." `

And lastly, trying to load with the second approach, I once again get this: 
`    from .neuralcoref import NeuralCoref
  File "cymem.pxd", line 1, in init en_coref_md.neuralcoref.neuralcoref
ValueError: cymem.cymem.Pool has the wrong size, try recompiling. Expected 64, got 48`

I also have tried `pip install --upgrade --force-reinstall --no-cache-dir MODEL_URL` (as suggested [here](https://github.com/huggingface/neuralcoref/issues/66) ), in both environments but with no luck.

What can I do to make it work? Any suggestions? 
The "factory error" seems like something I can do a work around with but I have no idea how. 

I have no idea what might be wrong and I'd LOVE to test out the package.

**Comment by JohnGiorgi** [comment]
Hi, 

Also on macOS Mojave, and had the same error:

```
from .neuralcoref import NeuralCoref
File "cymem.pxd", line 1, in init en_coref_md.neuralcoref.neuralcoref
ValueError: cymem.cymem.Pool has the wrong size, try recompiling. Expected 64, got 48
```

Here is what worked for me

```
conda create -n test python=3.6
source activate test
pip install https://github.com/huggingface/neuralcoref-models/releases/download/en_coref_md-3.0.0/en_coref_md-3.0.0.tar.gz
pip install space==2.0.13
```

I use `conda` but I don't think using `virtualenv` will change much. For me, it was about downgrading spacy to 2.0.13. After this, I can run the following without error

```python
import spacy
import en_coref_md
nlp = en_coref_md.load()
```

Hopefully that helps!

**Comment by thomwolf** [comment]
This should all be fixed in the new release (4.0).
With the simpler dependency structure of SpaCy 2.1+, it should be easier to the installation straightforward in the future.

**Comment by kkaraoglan** [comment]
I tried to install xyz following the installation instructions. I am getting the following errors please help?

  File "<input>", line 1, in <module>
  File "C:\Program Files\JetBrains\PyCharm 2019.3\plugins\python\helpers\pydev\_pydev_bundle\pydev_umd.py", line 197, in runfile
    pydev_imports.execfile(filename, global_vars, local_vars)  # execute the script
  File "C:\Program Files\JetBrains\PyCharm 2019.3\plugins\python\helpers\pydev\_pydev_imps\_pydev_execfile.py", line 18, in execfile
    exec(compile(contents+"\n", file, 'exec'), glob, loc)
  File "C:/Users/kkaraoglan/PycharmProjects/retina-sdk.py/tests/integration/CorefRes.py", line 2, in <module>
    nlp = spacy.load('en')
  File "C:\Users\kkaraoglan\AppData\Roaming\Python\Python37\site-packages\spacy\__init__.py", line 30, in load
    return util.load_model(name, **overrides)
  File "C:\Users\kkaraoglan\AppData\Roaming\Python\Python37\site-packages\spacy\util.py", line 172, in load_model
    return load_model_from_path(Path(name), **overrides)
  File "C:\Users\kkaraoglan\AppData\Roaming\Python\Python37\site-packages\spacy\util.py", line 198, in load_model_from_path
    meta = get_model_meta(model_path)
  File "C:\Users\kkaraoglan\AppData\Roaming\Python\Python37\site-packages\spacy\util.py", line 253, in get_model_meta
    raise IOError(Errors.E053.format(path=meta_path))
**OSError: [E053] Could not read meta.json from en\meta.json**
